README.md

```diff
@@ -6,8 +6,20 @@ Let's see a example from template.json
 
 {
   "name": "b2g-info cmd",
+  "type": "",
```

Comment:
suggest we can give a sample here.
for example: "type": "moz_minions.minions.ShellMinion",